doc/dev/for-khmer-developers.rst

```diff
@@ -50,15 +50,16 @@ First, never merge something unless it's been through a review!  This
 rule can be broken under specific conditions when doing a release; see
 :doc:`release`.
 
-Second, need to force another continuous integration run? Put "test
-this please" in a comment.  This is used to ask our continuous
-integration system to run on someone else's pull request -- by
-default, it only runs on commits from people who have write privileges
-to khmer, so you may need to do this if you're reviewing someone else's
-pull request.
-
-Third, we ask that all contributors set up standing Pull Requests
+Second, we ask that all contributors set up standing Pull Requests
```

Comment:
Currently this spells out Pull Request several times, and then uses the abbreviation PR several times. Perhaps we should follow the convention of introducing the abbreviation after the first occurrence in a section, and then using the abbreviation for all subsequent occurrences.